plugins/inputs/ceph/README.md

```diff
@@ -168,32 +168,52 @@ All admin measurements will have the following tags:
 
 ## Cluster
 
+- ceph_fsmap
+  - fields:
+    - up (float)
+    - in (float)
+    - max (float)
+    - up_standby (float)
+
 - ceph_health
   - fields:
-    - status
-    - overall_status
+    - status (string)
+    - status_code (int)
+
+- ceph_monmap
+  - fields:
+    - num_mons (float)
 
 - ceph_osdmap
   - fields:
     - epoch (float)
     - num_osds (float)
     - num_up_osds (float)
     - num_in_osds (float)
-    - full (bool)
-    - nearfull (bool)
     - num_remapped_pgs (float)
 
 - ceph_pgmap
   - fields:
     - version (float)
     - num_pgs (float)
+    - num_pools (float)
+    - num_objects (float)
     - data_bytes (float)
     - bytes_used (float)
     - bytes_avail (float)
     - bytes_total (float)
+    - inactive_pgs_ratio (float)
+    - degraded_objects (float)
+    - degraded_total (float)
+    - degraded_ratio (float)
+    - recovering_objects_per_sec (float)
+    - recovering_bytes_per_sec (float)
+    - recovering_keys_per_sec (float)
+    - num_objects_recovered (float)
+    - num_bytes_recovered (float)
+    - num_keys_recovered (float)
     - read_bytes_sec (float)
     - write_bytes_sec (float)
-    - op_per_sec (float, exists only in ceph <10)
```

Comment:
Same remark here, I would leave it for people still using these versions.

Reply:
Great, you re-added them, but if you did at the same place in the file. It wouldn't even be a visible change..

Reply:
Re-added with latest updates

Reply:
Yes, but I would keep it on the same line as before. (so between `write_bytes_sec` and `read_op_per_sec`)

Otherwise, I would opt to sort all list items alphabetically.

Reply:
Latest changes have all fields sorted alphabetically.

Previously, they matched the ordering of what was already in the plugin, or the output of the `ceph` commands, but this is probably easier overall going forward.